Casks/a/abbyy-finereader-pdf.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url "https://www.abbyy.com/finereader-pdf-mac-downloads/"
-    regex(%r{(?:Part #:.*?\n<td>)(\d+(?:[./]\d+)+)}i)
+    regex(%r{(?:Part #:.*?\n.*?<td>)(\d+(?:[./]\d+)+)}i)
```

Comment:
```suggestion
    regex(%r{>\s*Part\s+#:.*?<td>\s*v?(\d+(?:[./]\d+)+)\s*<}im)
```

We can resolve this issue while cleaning this up a bit at the same time:

* There's no reason for us to be using a [non-capturing] group around the "Part #:" text here (i.e., we're not doing something like `(?:this)?`, `(?:this)*?`, etc.), so we can get rid of the grouping
* I've tightened up the leading/trailing anchors a bit
* I've made the regex a bit more flexible in terms of whitespace
* I've added the `m` (multiline) flag, so the `.*?` after the "Part #:" text will simply match anything until the next `<td>`. It's a looser solution and that could go either way (i.e., it could keep the regex from breaking due to small content changes but it could also lead to an incorrect match due to the increased looseness) but there's maybe something to be said for the simplicity/flexibility.